functions/common/types/response.ts

```diff
@@ -4,7 +4,9 @@ export enum StatusCodes {
   "badRequest" = 400
 };
 
-export type Response<ResponseBody> = {
+export type Response = {
+  isBase64Encoded: boolean,
+  headers: {},
   statusCode: StatusCodes,
-  body: ResponseBody,
+  body: string,
```

Comment:
The body response has to be string, so any object that we will reply back has to be stringified so we won't be able to type check the response in the lambda. This strinfified was already happening behind the scenes on the integration response.